recipes/biorbd-casadi/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set libname = "biorbd" %}
+{% set name = "biorbd-casadi" %}
+{% set version = "1.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyomeca/{{ libname }}/archive/Release_{{ version }}.tar.gz
+  sha256: 7682fa0c34d6c9a4415c8293d12089845390516845334dbb5c4fcaedb60872ef
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py<36 or py>37]
```

Comment:
Does this need to this restrictive? If it does, `py!=37` would be easier to read.

Reply:
The restriction comes from CasADi which is not accessible yet for Windows (but I will fix that in the upcoming weeks), and will not compile with Python38 API (I don't have control for this since it depends on the core which I am not programmer on)